src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -518,32 +516,40 @@ function updateConstSumMessageQn(qnNum) {
     let allSame = true;
     let allNotNumbers = true;
     let answerSet = {};
+    const repeatedPoints = [];
 
     function fillWithZeroIfEmpty(inputFieldElement) {
         if (Number.isNaN(parseInt(inputFieldElement.val(), 10))) {
             inputFieldElement.val(0);
         }
     }
 
-    function checkAndDisplayMessage(messageElement) {
+    function checkAndDisplayMessage(instructionElement, messageElement) {
+        let instruction = '';
         let message = '';
+        const infoIcon = '<span class="glyphicon glyphicon-info-sign padding-right-10px"></span>';
+        const approvedIcon = '<span class="glyphicon glyphicon-ok padding-right-10px"></span>';
+        const errorIcon = '<span class="glyphicon glyphicon-remove padding-right-10px"></span>';
+
+        instruction += `${infoIcon} Total points distributed should add up to ${points}.<br>`;
+        if ((forceUnevenDistribution && !forceSomeUnevenDistribution) || forceAllUnevenDistribution) {
+            instruction += `${infoIcon} Every ${distributeToRecipients ? 'recipient' : 'option'} `
+                        + 'should be allocated different number of points.<br>';
+        }
+        if (forceSomeUnevenDistribution) {
+            instruction += `${infoIcon} At least one ${distributeToRecipients ? 'recipient' : 'option'} `
+                        + 'should be allocated different number of points.<br>';
+        }
+        instructionElement.addClass('text-color-blue');
+        instructionElement.removeClass('text-color-red');
+        instructionElement.removeClass('text-color-green');
 
-        if (allNotNumbers) {
-            message = `Please distribute ${points} points among the above `
-                    + `${distributeToRecipients ? 'recipients' : 'options'}.`;
-            messageElement.addClass('text-color-blue');
-            messageElement.removeClass('text-color-red');
-            messageElement.removeClass('text-color-green');
-        } else if (remainingPoints === 0) {
-            if (allowEvenDistribution || !forceUnevenDistribution
-                    || (forceUnevenDistribution && !forceSomeUnevenDistribution && allUnique)
-                    || (forceAllUnevenDistribution && allUnique)
-                    || (forceSomeUnevenDistribution && !allSame)) {
-                message = 'All points distributed!';
+        if (!allNotNumbers) {
```

Comment:
just saying if there are a lot of double negations for this variable, then you can consider changing the variable name to prevent such double negation.